packages/webapp/src/views/common/file-upload-check.html

```diff
@@ -39,13 +39,13 @@
             items: [
                 {
                     value: "yes",
-                    text: "Yes",
+                    text: "Yes, my local planning authority have approved this file",
```

Comment:
This is a common code file that is shared across all uploads, therefore this text change will affect all usage including landowner journey.  Please undo and ensure the text change only applies for the developer metric upload

Reply:
If continuing to use the common partial you could perhaps set the text value as a parameter, that in the common file defaults to "Yes", but you can override with a value from the view that pulls in the partial